### PR TITLE
[keymgr/dv] Update keymgr DV doc

### PR DESCRIPTION
**Pull request by weicaiyang**
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

File: hw/ip/keymgr/doc/dv/index.md [hunk 1]
```diff
@@ -73,28 +74,29 @@ The `keymgr_base_vseq` virtual sequence is extended from `cip_base_vseq` and ser
...
 ### Self-checking strategy
 #### Scoreboard
 The `keymgr_scoreboard` is primarily used for end to end checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
-* analysis port1:
-* analysis port2:
-<!-- explain inputs monitored, flow of data and outputs checked -->
+* tl_a_chan_fifo: An analysis FIFO to hold transactions from TL address channel.
+* tl_d_chan_fifo: An analysis FIFO to hold transactions from TL data channel.
+* req_fifo: An analysis FIFO to hold request data sended to KMAC.
```
Comment (cindychip) [inline hw/ip/keymgr/doc/dv/index.md:91]: sended -> sent

Comment (weicaiyang) [reply hw/ip/keymgr/doc/dv/index.md]: fixed, thanks Cindy.

**Comment by weicaiyang** [comment]
@sriyerg please take a look at this update when you have a chance. Thanks

File: hw/ip/keymgr/doc/dv/index.md [hunk 2]
```diff
@@ -73,28 +74,29 @@ The `keymgr_base_vseq` virtual sequence is extended from `cip_base_vseq` and ser
...
 #### Scoreboard
 The `keymgr_scoreboard` is primarily used for end to end checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
-* analysis port1:
-* analysis port2:
-<!-- explain inputs monitored, flow of data and outputs checked -->
+* tl_a_chan_fifo: An analysis FIFO to hold transactions from TL address channel.
+* tl_d_chan_fifo: An analysis FIFO to hold transactions from TL data channel.
+* req_fifo: An analysis FIFO to hold request data sent to KMAC.
+* rsp_fifo: An analysis FIFO to hold response digests recevied from KMAC.
```
Comment (matutem) [inline hw/ip/keymgr/doc/dv/index.md:92]: recevied -> received

**Review by matutem — APPROVED** [review]
LGTM, with one suggestion.

File: hw/ip/keymgr/doc/dv/index.md (hunk 2 above)
Comment (weicaiyang) [reply hw/ip/keymgr/doc/dv/index.md]: Good catch. Fixed.

**Pull request merged**